config/routes.rb

```diff
@@ -98,6 +98,8 @@
   namespace :school do
     get '/', to: 'home#show', as: :home
     get '/details', to: 'details#show', as: :details
+    get '/chromebooks/edit', to: 'chromebooks#edit', as: :edit_chromebooks_details
```

Comment:
If you don't set the `as:` clause, you will get a `school_chromebooks_edit_path` generated for you automatically as per Rails conventions.

Reply:
thanks!